riot/riot.py

```diff
@@ -393,6 +393,9 @@ def create_base_venv(pyversion, path=None, recreate=True):
     py_ex = f"python{pyversion}"
     py_ex = shutil.which(py_ex)
 
+    if not py_ex:
+        py_ex = shutil.which(pyversion):
```

Comment:
I am not so sure about this.... because if you are asking for `python 3.8` and it isn't found, we are just going to use `python 2.7` because it is available?

Reply:
So all this does is if like `python3.8` isn't available then it's going to check for `3.8`.

Presumably `pythonmypy` isn't going to exist so then we'd try `mypy`. Doing the check in the opposite order probably makes more sense though.